FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -555,17 +607,25 @@ void prvDefenderDemoTask( void * pvParameters )
 {
     bool xStatus = false;
     BaseType_t xExitStatus = EXIT_FAILURE;
-    uint32_t ulReportLength = 0, i, ulMqttSessionEstablished = 0;
+    uint32_t ulReportLength = 0UL, i, ulMqttSessionEstablished = 0UL;
 
     /* Remove compiler warnings about unused parameters. */
     ( void ) pvParameters;
 
     /* Start with report not received. */
     xReportStatus = ReportStatusNotReceived;
 
-    /* Set a report Id to be used. */
+    /* Set a report Id to be used.
+     *
+     * !!!NOTE!!!
+     * This demo just sets the report ID to 1, which should not be used in
+     * production. The report ID needs to be unique per report sent with a
+     * given Thing. We reccomend using a increasing unique id such as the
+     * current timestamp. */
```

Comment:
This is a good note. Can you please also add the same note to the demo description at the beginning of this file?